openhands_cli/refactor/widgets/input_field.py

```diff
@@ -0,0 +1,148 @@
+from typing import ClassVar
+
+from textual.binding import Binding
+from textual.containers import Container
+from textual.message import Message
+from textual.signal import Signal
+from textual.widgets import Input, TextArea
+
+from openhands_cli.refactor.core.commands import COMMANDS
+from openhands_cli.refactor.widgets.autocomplete import EnhancedAutoComplete
+
+
+class InputField(Container):
+    BINDINGS: ClassVar = [
+        Binding("f1", "toggle_input_mode", "Toggle single/multi-line input"),
```

Comment:
F* keys are typically very hard to use in mac os (fn on the botton left which is awkward for hand gesture), have we considered replacing these w/ something like `ctrl+i`?